e107_languages/English/admin/lan_language.php

```diff
@@ -56,11 +56,11 @@
 define("LANG_LAN_135", "Overwriting ");
 define("LANG_LAN_136", "Couldn't overwrite ");
 define("LANG_LAN_137", "Processed");
-define("LANG_LAN_140", "Hold down CTRL key to select multiple.[br]e.g. To check [b]lan_signup.php[/b] you'll want to also select [b]signup_shortcodes.php[/b] and [b]signup_template.php[/b].");
+define("LANG_LAN_140", "Hold down CTRL key to select multiple.<br />e.g. To check <b>lan_signup.php</b> you'll want to also select <b>signup_shortcodes.php</b> and <b>signup_template.php</b>.");
```

Comment:
According to [this issue](https://github.com/e107inc/e107/issues/6), HTML tags are not allowed in LANs. You have to use BBCodes instead. Then you can use the parser to get HTML formatted text:
```php
e107::getParser()->toHTML(LANG_LAN_140, true);
```